### PR TITLE
docs: Add guide for deploying a Bun app to Render

### PR DESCRIPTION
**Pull request by jess-render**
### What does this PR do?

Adds a guide that shows how to deploy a Bun app on Render.

- [x] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [ ] Code changes

### How did you verify your code works?
Manually tested by following the instructions

### CODE REVIEW

**Comment by jess-render** [comment]
cc @Jarred-Sumner (since I don't seem to have permission to add a Reviewer or Assignee)

File: docs/guides/ecosystem/render.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+In your [Render Dashboard](https://dashboard.render.com/), click `New` > `Web Service` and connect your `myapp` repo.
+
+---
+
+In the Render UI, provide the following values during web service creation:
+
+|             |           |
+| ----------- | --------- |
+| **Runtime** | `Node` |
```
Comment (gvilums) [inline docs/guides/ecosystem/render.md:69]: Is `Node` the right value for `Runtime`?

Comment (jess-render) [reply docs/guides/ecosystem/render.md:69]: Yes -- we don't have a separate one for Bun currently. Both Bun and Node are available in the "Node" runtime. 

We considered naming it "Node / Bun" but some teammates thought that might be confusing at the moment.

Comment (gvilums) [reply docs/guides/ecosystem/render.md:69]: Makes sense 👍

**Review by gvilums — APPROVED** [review]

**Pull request merged**